src/promptflow/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.7.0 (Upcoming)
 
+### NOTICES
+- Import warning will be print when importing from `promptflow` namespace, please use imports from new namespaces
```

Comment:
```suggestion
- Import warning will be printed when importing from `promptflow` namespace, please use imports from new namespaces
```

Reply:
honestly, i'm not sure if it is right, just read this more smoothly...

Reply:
Half correct for you and me, print -> printed, and 'importing' was correct.